src/main/java/nextstep/courses/domain/ImageType.java

```diff
@@ -1,9 +1,18 @@
 package nextstep.courses.domain;
 
+import java.util.Arrays;
+
 public enum ImageType {
     GIF,
     JPG,
     JPEG,
     PNG,
-    SVG
+    SVG;
+
+    public static ImageType of(String name) {
+        return Arrays.stream(values())
+                .filter(it -> it.name().equals(name))
```

Comment:
대소문자 구분없이 비교해도 좋을 것 같아요
```suggestion
                .filter(it -> it.name().equalsIgnoreCase(name))
```